torch/linalg/__init__.py

```diff
@@ -2378,3 +2378,63 @@
     >>> torch.dist(Q.mT @ Q, torch.eye(4))
     tensor(6.2158e-07)
 """)
+
+vander = _add_docstr(_linalg.linalg_vander, r"""
+vander(x, N=None, increasing=False) -> Tensor
```

Comment:
This should have `increasing=True`

Reply:
(or remove the increasing argument all together)